### ISSUE TITLE
Strange shadows differences with BFG?

### ISSUE THREAD

**Issue by Hexaae** [MadDeCoDeR/Classic-RBDOOM-3-BFG#63]
I noticed a strange difference between DOOM 3 BFG and the same "base" files used with DOOM BFA:

Doom 3 BFG:

![w2us3m0trqw61](https://user-images.githubusercontent.com/8591462/116828797-50495680-aba1-11eb-8217-53ef9ccc9f6b.jpg)

Doom 3 BFA (using D3 BFG's base files):
![DOOMBFA Screenshot 2021 05 02 - 23 07 54 68](https://user-images.githubusercontent.com/8591462/116828814-74a53300-aba1-11eb-93b8-129480df138e.png)

Is this a bug or what?

**Comment by MadDeCoDeR** [comment]
Are you still using soft shadows right?

**Comment by Hexaae** [comment]
Yes of course. Try yourself start a new game and enter the bar... With DOOM BFA some shadows are missing. They are shown using same files with GOG exe but RBDOOM3BFG exe has thesame bug...

**Comment by SEAGitUser** [comment]
That's because geometry does not have back sides( facing to the light ). Go to noclip mode, then fly outside to see what's happening.

**Comment by Hexaae** [comment]
Weird. Original GOG exe and MS Store version do cast shadows in the bar
though...

Il giorno lun 3 mag 2021 alle ore 14:24 S.E.A. ***@***.***>
ha scritto:

**Comment by motorsep** [comment]
I don't think it has anything to do with backside polygons. Look at the characters sitting at the table. No shadows from them either.

**Comment by Hexaae** [comment]
> I don't think it has anything to do with backside polygons. Look at the characters sitting at the table. No shadows from them either.

Correct. I noticed the same.

**Comment by Arl90** [comment]
I just point at this here: https://github.com/RobertBeckebans/RBDOOM-3-BFG/issues/622

**Comment by MadDeCoDeR** [comment]
> I just point at this here: [RobertBeckebans/RBDOOM-3-BFG#622](https://github.com/RobertBeckebans/RBDOOM-3-BFG/issues/622)

FYI it's called r_shadowMapRegularDepthBiasScale

**Comment by Arl90** [comment]
> > I just point at this here: [RobertBeckebans/RBDOOM-3-BFG#622](https://github.com/RobertBeckebans/RBDOOM-3-BFG/issues/622)
> 
> FYI it's called r_shadowMapRegularDepthBiasScale

Oh, thanks! I'll do an edit.

**Comment by Hexaae** [comment]
> I just point at this here: [RobertBeckebans/RBDOOM-3-BFG#622](https://github.com/RobertBeckebans/RBDOOM-3-BFG/issues/622)
Not exactly the same when using 0.9999. Look at the original game pics above in this thread: also tables and chairs cast shadows, still missing in your pics with 0.9999
